web/skins/modern/accountstatus/accountstatus.php

```diff
@@ -79,7 +79,7 @@
                         <?php
                         switch ($statusInfo['errorcode']) {
                             case "GENERATOR_CONSUMED":
-                                echo $uiElements->boxError(_("You attempted to download an installer that was already downloaded before. Please request a new token from your administrator instead."), _("Attempt to re-use download link"), TRUE) . "<p>";
+                                echo $uiElements->boxError(_("You attempted to download an installer that was already downloaded before. Please request a new token from your administrator instead."), _("Attempt to reuse download link"), TRUE) . "<p>";
```

Comment:
This one is debatable, but `reuse` is in all dictionaries, why write `re-use` instead?

Reply:
This string is now translated into many languages as-is, and any change invalidates the translations.

Reply:
Translations need not change, they are not invalidated.

Reply:
gettext *will* change the status of translation into untranslated, fuzzy, because the string is not identical.

Reply:
Wouldn't this be an issue in the way you use gettext? I have already changed all pot/po files accordingly.

Reply:
Oh, I overlooked that you also changed the translations, sorry.